ui5/browser/controller/Browser.controller.js

```diff
@@ -586,6 +638,7 @@ sap.ui.define(['sap/ui/core/Component',
       
       /** process initial message, now it is list of existing canvases */
       processInitMsg: function(msg) {
+         this.websocket.Send('WORKDIR:'); // Update the breadcrumbs
```

Comment:
Better use message "GETWORKDIR" here. Need correspondent change in c++

Reply:
Fixed in : bb144ba